Makefile

```diff
@@ -74,8 +76,10 @@ release-cp: release-dir
 	cp -a tuned.py tuned.spec tuned.service tuned.tmpfiles Makefile tuned-adm.py \
 		tuned-adm.bash dbus.conf recommend.conf tuned-main.conf 00_tuned \
 		92-tuned.install bootcmdline modules.conf com.redhat.tuned.policy \
-		tuned-gui.py tuned-gui.glade tuned-ppd.py \
-		tuned-gui.desktop functions compile_plugin_docs.py $(VERSIONED_NAME)
+		tuned-ppd.py functions compile_plugin_docs.py $(VERSIONED_NAME)
+	ifeq ($(INSTALL_GUI),1)
+		cp -a tuned-gui.py tuned-gui.glade tuned-gui.desktop $(VERSIONED_NAME)
+	endif
```

Comment:
Oh, this is gonna break hard. `release-cp` is the target for creating the release tarball, it should NOT be conditional on `INSTALL_GUI` or anything else.